{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2021-30470",
  "PublicDate": "2021-04-11T00:00:00Z",
  "References": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-30470",
    "https://sourceforge.net/p/podofo/tickets/130/"
  ],
  "Description": "[Unknown description]",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": null,
  "Priority": "medium",
  "DiscoveredBy": "",
  "AssignedTo": "",
  "Patches": {
    "libpodofo": {
      "bionic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "groovy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "hirsute": {
        "Status": "needs-triage",
        "Note": ""
      },
      "precise/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "trusty": {
        "Status": "ignored",
        "Note": "out of standard support"
      },
      "trusty/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      },
      "xenial": {
        "Status": "needs-triage",
        "Note": ""
      }
    }
  },
  "UpstreamLinks": {}
}